pkgs/development/libraries/libliftoff/default.nix

```diff
@@ -1,9 +1,9 @@
-{ callPackage, fetchFromGitLab, ... }:
+{ callPackage, fetchFromGitLab, fetchpatch }:
 let
   mkVariant =
-    version: hash:
+    version: hash: patches:
```

Comment:
3 positional arguments is starting to get a bit unwieldy. Can you refactor into named arguments? `patches` can default to the empty list then.

Reply:
Switched to named arguments.